dlp/src/main/java/com/example/dlp/StoredInfoTypes.java

```diff
@@ -0,0 +1,346 @@
+                  System.out.println("\tCreated: " + currentVersion.getCreateTime());
+                  System.out.println("\tState: " + currentVersion.getState());
+                  System.out.println("\tError count: " + currentVersion.getErrorsCount());
+                }
+                if (storedInfoType.getPendingVersionsCount() > 0) {
+                  System.out.println("Pending versions:");
+                  for (StoredInfoTypeVersion pendingVersion :
+                      storedInfoType.getPendingVersionsList()) {
+                    System.out.println("\tCreated:" + pendingVersion.getCreateTime());
+                    System.out.println("\tState: " + pendingVersion.getState());
+                    System.out.println("\tError count: " + pendingVersion.getErrorsCount());
+                  }
+                }
+              });
+    } catch (Exception e) {
+      System.out.println("Error listing stored infoTypes :" + e.getMessage());
+    }
+  }
+  // [END dlp_list_stored_info_types]
+
+  // [START dlp_delete_stored_info_type]
+  /**
+   * Delete a DLP stored infoType in a project.
+   *
+   * @param projectId The project ID to run the API call under.
+   * @param storedInfoTypeId Stored infoType ID
+   */
+  private static void deleteStoredInfoType(String projectId, String storedInfoTypeId) {
+
+    String storedInfoTypeName =
+        ProjectStoredInfoTypeName.of(projectId, storedInfoTypeId).toString();
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+      DeleteStoredInfoTypeRequest deleteStoredInfoTypeRequest =
+          DeleteStoredInfoTypeRequest.newBuilder().setName(storedInfoTypeName).build();
+      dlpServiceClient.deleteStoredInfoType(deleteStoredInfoTypeRequest);
+
+      System.out.println("Stored infoType deleted: " + storedInfoTypeName);
+    } catch (Exception e) {
+      System.out.println("Error deleting stored infoType :" + e.getMessage());
+    }
+  }
+  // [END dlp_delete_stored_info_type]
+
+  /** Command line application to create, list and delete stored infoTypes. */
+  public static void main(String[] args) throws Exception {
+
+    OptionGroup optionsGroup = new OptionGroup();
+    optionsGroup.setRequired(true);
+
+    Option createStoredInfoTypeOption =
+        new Option("c", "create", false, "Create stored infoType from a list of words or phrases");
+    optionsGroup.addOption(createStoredInfoTypeOption);
+
+    Option listStoredInfoTypesOption = new Option("l", "list", false, "List stored infoTypes");
+    optionsGroup.addOption(listStoredInfoTypesOption);
+
+    Option deleteStoredInfoTypeOption = new Option("d", "delete", false, "Delete stored infoType");
+    optionsGroup.addOption(deleteStoredInfoTypeOption);
+
+    Options commandLineOptions = new Options();
+    commandLineOptions.addOptionGroup(optionsGroup);
+
+    Option gcsInputFilePathOption =
+        Option.builder("gcsInputFilePath").hasArg(true).required(false).build();
```

Comment:
I would prefer to see these in the typical builder style then the line is too long:
```java
Option.builder("gcsInputFilePath")
.hasArg(true)
.required(false)
.build();
```